bitnami/sonarqube/README.md

```diff
@@ -310,18 +310,18 @@ The command removes all the Kubernetes components associated with the chart and
 | -------------------------------------- | ---------------------------------------------------------------------------------- | ------------------- |
-| `postgresql.enabled`                   | Deploy PostgreSQL subchart                                                         | `true`              |
-| `postgresql.nameOverride`              | Override name of the PostgreSQL chart                                              | `""`                |
-| `postgresql.auth.existingSecret`       | Existing secret containing the password of the PostgreSQL chart                    | `""`                |
-| `postgresql.auth.password`             | Password for the postgres user of the PostgreSQL chart (auto-generated if not set) | `""`                |
-| `postgresql.auth.username`             | Username to create when deploying the PostgreSQL chart                             | `bn_sonarqube`      |
-| `postgresql.auth.database`             | Database to create when deploying the PostgreSQL chart                             | `bitnami_sonarqube` |
-| `postgresql.service.ports.postgresql`  | PostgreSQL service port                                                            | `5432`              |
-| `postgresql.persistence.enabled`       | Use PVCs when deploying the PostgreSQL chart                                       | `true`              |
-| `postgresql.persistence.existingClaim` | Use an existing PVC when deploying the PostgreSQL chart                            | `""`                |
-| `postgresql.persistence.storageClass`  | storageClass of the created PVCs                                                   | `""`                |
-| `postgresql.persistence.accessMode`    | Access mode of the created PVCs                                                    | `ReadWriteOnce`     |
-| `postgresql.persistence.size`          | Size of the created PVCs                                                           | `8Gi`               |
+| `postgresql.primary.enabled`             | Deploy PostgreSQL subchart                                                         | `true`              |
+| `postgresql.primary.nameOverride`        | Override name of the PostgreSQL chart                                              | `""`                |
+| `postgresql.primary.auth.existingSecret` | Existing secret containing the password of the PostgreSQL chart                    | `""`                |
+| `postgresql.primary.auth.password`       | Password for the postgres user of the PostgreSQL chart (auto-generated if not set) | `""`                |
+| `postgresql.primary.auth.username`       | Username to create when deploying the PostgreSQL chart                             | `bn_sonarqube`      |
+| `postgresql.primary.auth.database`       | Database to create when deploying the PostgreSQL chart                             | `bitnami_sonarqube` |
```

Comment:
These variables need not be changed if soy check the chart of postgresql does not appear below primary